projects/packages/patchwork-redefine-exit/composer.json

```diff
@@ -45,7 +45,7 @@
 	"extra": {
 		"autotagger": true,
 		"branch-alias": {
-			"dev-trunk": "0.1.x-dev"
+			"dev-trunk": "1.0.x-dev"
```

Comment:
Do we want `.x` here?

Reply:
Looks like that's the convention, yes.

Reply:
Yes, I think it's even required. See https://getcomposer.org/doc/articles/aliases.md for details.